src/golang/detectors/bad-tls-settings/BadTlsSettings.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"crypto/tls"
+	"fmt"
+	"net/http"
+)
+
+// {fact rule=insecure-cryptography@v1.0 defects=1}
```

Comment:
Are we going to remove these tags?

Reply:
we cann't remove these tags. 
those tags is used internal to extract a rules compliant or noncompliant examples